node_modules/oae-mixpanel/lib/api.js

```diff
@@ -14,24 +14,132 @@
+    }
+
+    // Request options object
+    var data = {
+        'from_date': _getStartDate(Date.now()),
+        'to_date': moment.utc(Date.now()).format('YYYY-MM-DD'),
+        'event': 'userLoggedOut',
+        'on': 'properties["tenant"]',
+        'type': 'unique'
+    };
+
+    // Return the unique users for a specific tenant if the endpoint was accessed on the tenant router
+    if (ctx.tenant().alias !== 'admin') {
+        data = _.extend(data, {
+            'where': '"' + ctx.tenant().alias + '" == properties["tenant"]'
+        });
+    }
+
+    // Request the data from the MixPanel API
+    exporter.segmentation(data, function(data) {
+        if (data.error) {
+            return callback({'code': 400, 'msg': data.error})
+        }
+
+        return callback(null, data);
+    });
+};
+
+/**
+ * Get the unique user count for the current month for a specific tenant
+ *
+ * @param  {Context}    ctx                 Standard context object containing the current user and the current tenant
+ * @param  {String}     alias               The alias of the tenant to get the stats for
+ * @param  {Function}   callback            Standard callback function
+ * @param  {Object}     callback.err        An error that occurred, if any
+ * @param  {Object}     callback.stats      Object containing the stats for the tenant
+ */
+var getUniqueUsersForTenant = module.exports.getUniqueUsersForTenant = function(ctx, alias, callback) {
+    if (ctx.user() && !ctx.user().isAdmin()) {
+        return callback({'code': 401, 'msg': 'Only administrators can request stats'});
+    }
+
+    // Perform some basic validation
+    var validator = new Validator();
+    validator.check(alias, {'code': 400, 'msg': 'Missing alias'}).notEmpty();
+    validator.check(alias, {'code': 400, 'msg': 'The tenant alias should not contain a space'}).notContains(' ');
+    validator.check(alias, {'code': 400, 'msg': 'The tenant alias should not contain a colon'}).notContains(':');
+    if (validator.hasErrors()) {
+        return callback(validator.getFirstError());
+    }
+
+    // Request options object
+    var data = {
+        'from_date': _getStartDate(Date.now()),
+        'to_date': moment.utc(Date.now()).format('YYYY-MM-DD'),
+        'event': 'userLoggedIn',
+        'on': 'properties["tenant"]',
+        'where': '"' + alias + '" == properties["tenant"]',
+        'type': 'unique'
+    };
+
+    // Request the data from the MixPanel API
+    exporter.segmentation(data, function(data) {
+        if (data.error) {
+            return callback({'code': 400, 'msg': data.error})
+        }
+
+        return callback(null, data);
+    });
+};
+
+/**
+ * Return the start date
+ *
+ * @param  {Date}       [date]      The current date
+ * @return {String}                 The start date in ISO 8601 format (e.g. 2015-07-01)
+ * @api private
+ */
+var _getStartDate = function(date) {
+    date = date || Date.now();
```

Comment:
It looks like all the handling for "date" has been millis since epoch. Can rename `date` to `startMillis`?

Also, all places where we do `_getStartDate(Date.now());`, we can just change to `_getStartDate()` since it's the default.